tests/bluechi_test/systemctl.py

```diff
@@ -31,9 +32,29 @@ def start_unit(self, unit_name: str, check_result: bool = True, expected_result:
             -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
         return self._do_operation_on_unit(unit_name, "start", check_result, expected_result)
 
-    def stop_unit(self, unit_name: str, check_result: bool = True, expected_result: int = 0)  \
+    def stop_unit(self,
+                  unit_name: str,
+                  check_result: bool = True,
+                  expected_result: int = 0,
+                  operation_results: Set[str] = ("success", "exit-code"))  \
             -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
-        return self._do_operation_on_unit(unit_name, "stop", check_result, expected_result)
+        result, output = self._do_operation_on_unit(unit_name, "stop", check_result, expected_result)
+
+        if check_result:
+            time.sleep(1)
```

Comment:
I'd like to avoid this sleep if possible. When `systemctl stop` returns, the result should be set, right? So this sleep could be removed?

Reply:
It works even without it, removing